app/grandchallenge/algorithms/models.py

```diff
@@ -407,6 +407,98 @@ def get_jobs_limit(self, user):
                 credits_left = user_credit.credits
             return max(credits_left, 0) // max(self.credits_per_job, 1)
 
+    @property
+    def usage_chart_statuses(self):
+        """What statuses should be included on the chart"""
+        return [Job.SUCCESS, Job.CANCELLED, Job.FAILURE]
+
+    @property
+    def usage_chart_status_choices(self):
+        """A map of int to string for Job.choice"""
+        return {k: v for k, v in Job.status.field.choices}
+
+    @cached_property
+    def usage_statistics(self):
```

Comment:
You're not calculating user specific usage stats, as described in the pitch. Is there a security concern with that, or are you planning on adding that later?

Reply:
Will maybe add it later.